src/display/api.js

```diff
@@ -104,10 +104,16 @@ if (typeof PDFJSDev !== 'undefined' &&
  * @property {string} docBaseUrl - (optional) The base URL of the document,
  *   used when attempting to recover valid absolute URLs for annotations, and
  *   outline items, that (incorrectly) only specify relative URLs.
- * @property {boolean} disableNativeImageDecoder - (optional) Disable decoding
+ * @property {boolean} disableNativeImageDecoder - (deprecated) Disable decoding
  *   of certain (simple) JPEG images in the browser. This is useful for
  *   environments without DOM image support, such as e.g. Node.js.
  *   The default value is `false`.
+ * @property {string} nativeImageDecoderSupport - (optional) Strategy when
```

Comment:
Nit: I'm now wondering if it wouldn't suffice to just use the shorter `nativeImageDecoding` name here (and everywhere else in the patch)?